src/main/java/com/example/demoadminpanel/user/model/CreateUserRequest.java

```diff
@@ -4,41 +4,50 @@
 import com.example.demoadminpanel.user.model.enums.UserStatus;
 import lombok.Data;
 
+import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Pattern;
 import javax.validation.constraints.Size;
 
 @Data
 public class CreateUserRequest {
-    @NotNull(message = "Name should not be null")
-    @Size(min = 2, max = 25, message = "Name should be more than 2 and less than 25 characters")
+
+    @NotBlank(message = "name-Name should not be blank.")
+    @Size(min = 2, max = 25, message = "name-Name should be {min}-{max} characters long.")
     private String name;
 
-    @NotNull(message = "Username should not be null")
-    @Size(min = 2, max = 20, message = "Username should be more than 2 and less than 20 characters")
+    @NotBlank(message = "username-Username should not be blank.")
+    @Size(min = 2, max = 20, message = "username-Username should be {min}-{max} characters long.")
```

Comment:
Is the frontend supposed to split error message on hyphen? If yes, then hyphen in "{min}-{max}" will make it ambiguous and unable to be split or will be split incorrectly.